includes/Checker/AJAX_Runner.php

```diff
@@ -75,4 +75,15 @@ protected function get_check_slugs_param() {
 
 		return $checks;
 	}
+
+	/**
+	 * Returns the include experimental paramater based on the request.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return bool Returns true to include experimental checks else false.
```

Comment:
@mukeshpanchal27 That's temporary though, so better not to document it like that as it could change in the future. Also since this is an abstract method from a parent class, we shouldn't change the documentation here IMO.